src/misc/lv_event.c

```diff
@@ -41,6 +44,10 @@
     #define LV_TRACE_EVENT(...)
 #endif
 
+#define EVENT_MARK_DELETING(list, dsc) do { list->has_marked_deleting = true; (dsc)->filter |= LV_EVENT_MARKED_DELETING; } while(0)
+#define EVENT_IS_MARKED_DELETING(dsc) (((dsc)->filter & LV_EVENT_MARKED_DELETING) != 0)
```

Comment:
Please convert these to inline functions for better debug experience.

Reply:
Done